lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoClass.kt

```diff
@@ -72,23 +72,15 @@ class KoClass private constructor(private val ktClass: KtClass) : KoComplexDecla
 
     fun hasDataModifier() = ktClass.isData()
```

Comment:
Let's use `hasModifiers(KoModifier.X)` for every modifier property because "kt methods" may have additional behavior eg. `ktClass.isAbstract()`  is doing this 
```
fun KtClass.isAbstract(): Boolean = isInterface() || hasModifier(KtTokens.ABSTRACT_KEYWORD)
```